lib/main.dart

```diff
@@ -15,12 +18,16 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
-        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
-        useMaterial3: true,
+        extensions: const [AppTheme()],
+        colorScheme: const ColorScheme.light().copyWith(
+            surface: Colors.white,
+          ),
       ),
+      debugShowCheckedModeBanner: false,
       localizationsDelegates: AppLocalizations.localizationsDelegates,
       supportedLocales: AppLocalizations.supportedLocales,
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      //home: const MyHomePage(title: 'Flutter Demo Home Page'),
```

Comment:
what this comment is doing out here. Is this some legacy trash code or will it be usefull?